apps/storefront/src/pages/AccountSetting/index.tsx

```diff
@@ -132,6 +130,7 @@ function AccountSetting() {
 
         const { [key]: accountSettings } = await fn(params);
 
+        const accountFormAllFields = await getB2BAccountFormFields(isBCUser ? 1 : 2);
```

Comment:
"how many fields do you want? One, two..?" 😆

Reply:
I wish that was what it means...